content/millennium/r4/diagnostic/questionnaire-response.md

```diff
@@ -11,11 +11,21 @@ title: QuestionnaireResponse | R4 API
 
 ## Overview
 
-The QuestionnaireResponse resource is a collection of answers to a given questionnaire typically used to request patient healthcare information. An example of a questionnaire is a form used to collect a patient's social history. Currently, only social history is supported.
+The QuestionnaireResponse resource is a collection of answers to a given questionnaire typically used to request patient healthcare information. As long as the patient exists, a QuestionnaireReponse will be returned even when the data does not exist. A given QuestionnaireResponse must belong to a corresponding Questionnaire. An example of a questionnaire is a form used to collect a patient's social history. Currently, only social history is supported.
 
 The following fields are returned if valued:
 
 * [Id](https://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Questionnaire](https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.questionnaire){:target="_blank"}
```

Comment:
I think we should still include authored and texts in this list bc this rerfers to reads as well

Reply:
and keep them in the read and search examples, they are still returned

Reply:
https://github.com/cerner/fhir.cerner.com/pull/669/commits/9d32ffbd4f6e433064a04e94468fc74bd883cfe6

Reply:
dont put them back in the update example though

Reply:
Oh man, sorry about that - fixed: 1c8154bbdb58cf93f0a5a23ed377a74b4a4583b2